package.json

```diff
@@ -24,8 +24,8 @@
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn constraints --fix",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!**/CHANGELOG.md' '!**/CHANGELOG.old.md' '**/*.yml' '!.yarnrc.yml' '!merged-packages/**' --ignore-path .gitignore",
     "prepack": "./scripts/prepack.sh",
-    "prepare-preview-builds": "yarn workspaces foreach --parallel run prepare-manifest:preview",
-    "publish-previews": "yarn workspaces foreach --parallel run publish:preview",
+    "prepare-preview-builds": "scripts/prepare-preview-builds.sh",
```

Comment:
We let the script take care of applying changes across the monorepo instead of adding a script to each package and using `yarn workspaces foreach`. We do this because once we change a manifest we have to run `yarn install` in order to run another Yarn command. `yarn workspaces foreach` is not intelligent; it's more or less a shortcut for:

```
for each workspace in the monorepo
  yarn <whatever command you want to run>
end
```

And so if we use `yarn workspaces foreach`, the first package in the list of workspaces will get updated but none of the other packages will get updated because Yarn will fail.